README.md

```diff
@@ -14,7 +16,7 @@
 - The project aims to provide token standards, decentralized exchanges, and other utility functions that can be used on the ibet-Network.
 
 ## Dependencies
-- [python3](https://www.python.org/downloads/release/python-368/)
+- [Python3](https://www.python.org/downloads/release/python-368/)
```

Comment:
One of the following links would be better because Python 3.6.8 is not supported and Python 3.10 is used in `Dockerfile`.

> - https://www.python.org/downloads/
> - https://www.python.org/downloads/release/python-3108/

Reply:
🥺🥺🥺